content/reference/endpoints/rentals_fees.md

```diff
@@ -16,15 +17,41 @@ id               | Integer | Read       | Rentals Fee's id.
 -----------------|---------|------------|------------
 always_applied   | Boolean | Read       | Available to all seasons and periods if set to true.
 maximum_bookable | Integer | Read       | Rentals Fee's maximum booked count.
+name             | [Object](/reference/enums#formats)   | Read       | Rentals Fee's name, list of locales described in [enums section](/reference/enums#locales).
 public           | Boolean | Read       | Publicly bookable by client if set to true.
+rate             | [Decimal](/reference/enums#formats)  | Read       | Rentals Fee's rate.
+rate_kind        | String  | Read       | Rentals Fee's rate kind, list of fee's rate kinds described in [enums section](/reference/enums#fee-rate-kinds).
 required         | Boolean | Read       | Always included for new bookings, also public if set to true.
 -----------------|---------|------------|------------
+archived_at      | [Time](/reference/enums#formats) | Read       | Rentals Fee's archive time.
 created_at       | [Time](/reference/enums#formats) | Read       | Rentals Fee's create time.
 updated_at       | [Time](/reference/enums#formats) | Read       | Rentals Fee's update time.
 start_date       | [Date](/reference/enums#formats) | Read       | Rentals Fee's start date.
 end_date         | [Date](/reference/enums#formats) | Read       | Rentals Fee's end date.
 {: class="table table-bordered"}
   </div>
+  <div class="tab-pane" id="rentals_write" markdown="1">
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read       | Rentals Fee's id.
+fee_id           | Integer | Write      | Fee id related to the Rentals Fee.
+season_ids       | Array   | Write      | Season ids related to the Rentals Fee.
+-----------------|---------|------------|------------
+always_applied   | Boolean | Read/Write | Available to all seasons and periods if set to true.
+maximum_bookable | Integer | Read/Write | Rentals Fee's maximum booked count.
+name             | [Object](/reference/enums#formats)   | Read       | Rentals Fee's name, list of locales described in [enums section](/reference/enums#locales).
+public           | Boolean | Read       | Publicly bookable by client if set to true.
```

Comment:
on, on admin v2 it's using `status`, should be here as well then. Needed to set this.